core/storage/face/readable.hpp

```diff
@@ -27,6 +27,14 @@ namespace kagome::storage::face {
      */
     virtual outcome::result<V> get(const K &key) const = 0;
 
+    /**
+     * @brief Get value by key
+     * @param key K
+     * @return V if contains(K) or boost::none
+     */
+    virtual outcome::result<boost::optional<V>> tryGet(
```

Comment:
Why do we need this?

Reply:
Because it is more expressive since absence of a value is not an error here, it is totally expected behaviour.